### PR TITLE
[KYUUBI #3039][DOCS] Add operation resource REST docs

### PR DESCRIPTION
**Pull request by deadwind4**
### _Why are the changes needed?_
Add operation resource REST docs

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by pan3793 — APPROVED** [review]

File: docs/client/rest/rest_api.md [hunk 1]
```diff
@@ -261,6 +261,71 @@ Create an operation with GET_SCHEMAS type
 |:-----------|:----------------------------|:-------|
 | identifier | The identifier of operation | String |
 
+## Operation Resource
+
+### GET /operations/${operationHandle}/event
+
+Get an event of operation
```
Comment (yaooqinn) [inline docs/client/rest/rest_api.md:268]: Get the current event of the operation by the specified operation handle.

File: docs/client/rest/rest_api.md [hunk 2]
```diff
@@ -261,6 +261,71 @@ Create an operation with GET_SCHEMAS type
...
+
+#### Request Body
+
+| Name   | Description                                                                 | Type   |
+|:-------|:----------------------------------------------------------------------------|:-------|
+| action | The action that is applied the operation, for example: 'cancel' and 'close' | String |
+
+### GET /operations/${operationHandle}/resultsetmetadata
+
+Get a result set of metadata
```
Comment (yaooqinn) [inline docs/client/rest/rest_api.md:286]: Get the result set schema of the operation by the specified operation handle.

File: docs/client/rest/rest_api.md [hunk 3]
```diff
@@ -261,6 +261,71 @@ Create an operation with GET_SCHEMAS type
...
+
+#### Response Body
+
+| Name    | Description                 | Type               |
+|:--------|:----------------------------|:-------------------|
+| columns | The descriptions of columns | List of ColumnDesc |
+
+### GET /operations/${operationHandle}/log
+
+Get a set of operation logs
```
Comment (yaooqinn) [inline docs/client/rest/rest_api.md:296]: Get a list of operation log lines of the running operation by the specified operation handle.

File: docs/client/rest/rest_api.md [hunk 4]
```diff
@@ -261,6 +261,71 @@ Create an operation with GET_SCHEMAS type
...
+#### Response Body
+
+| Name      | Description              | Type            |
+|:----------|:-------------------------|:----------------|
+| logRowSet | The set of log set       | List of Strings |
+| rowCount  | The count of log row set | Int             |
+
+### GET /operations/${operationHandle}/rowset
+
+Get a set of result rows
```
Comment (yaooqinn) [inline docs/client/rest/rest_api.md:313]: Get the operation result as a list of rows by the specified operation handle.

File: docs/client/rest/rest_api.md [hunk 5]
```diff
@@ -261,6 +261,71 @@ Create an operation with GET_SCHEMAS type
...
+
+Get the current event of the operation by the specified operation handle.
+
+#### Response Body
+
+The [KyuubiOperationEvent](#kyuubioperationevent).
+
+### PUT /operations/${operationHandle}
+
+Put an action to the operation
```
Comment (yaooqinn) [inline docs/client/rest/rest_api.md:276]: Perform an action to the pending or running operation. Currently, supported actions are:

- Cancel: to cancel the operation, which means the operation and its corresponding background task will be stopped, and its state will be switched to CANCELED. A canceled operation's status can still be fetched by client requests.
- Close: to close the operation,  which means the operation and its corresponding background task will be stopped, and its state will be switched to CLOSED. A CLOSED operation's status will be removed on the server side and can not be fetched anymore.

File: docs/client/rest/rest_api.md [hunk 6]
```diff
@@ -261,6 +261,71 @@ Create an operation with GET_SCHEMAS type
...
+
+#### Request Body
+
+| Name   | Description                                                                 | Type   |
+|:-------|:----------------------------------------------------------------------------|:-------|
+| action | The action that is applied the operation, for example: 'cancel' and 'close' | String |
```
Comment (yaooqinn) [inline docs/client/rest/rest_api.md:282]: Let's clarify the action description a bit like the above comments.

Comment (deadwind4) [reply docs/client/rest/rest_api.md]: The preview results.
image

**Review by yaooqinn — APPROVED** [review]

**Comment by deadwind4** [comment]
@yaooqinn I have rebased and CI has passed.

**Comment by yaooqinn** [comment]
thanks, merged to master

**Pull request closed without merging**